Maple2.File.Parser/Xml/Table/IndividualItemDrop.cs

```diff
@@ -16,7 +16,7 @@ namespace Maple2.File.Parser.Xml.Table;
 // ./data/xml/table/na/individualitemdrop_gearbox.xml
 [XmlRoot("ms2")]
 public partial class IndividualItemDropRoot {
-    [M2dFeatureLocale(Selector = "individualDropBoxID")] private IList<IndividualItemDrop> _individualDropBox;
+    [M2dFeatureLocale(Selector = "individualDropBoxID|dropGroup|item")] private IList<IndividualItemDrop> _individualDropBox;
```

Comment:
do you need `item` here? That would mean there are multiple entries with the same `individualDropBoxID` and `dropGroup` but different `item`

Reply:
yeah there's multiple entires with individualDropBoxID and dropGroups with the same values.